### DISCUSSION TITLE
Uncaught ReferenceError: y is not defined

### DISCUSSION THREAD

**Discussion by khousuylong** [maplibre/maplibre-gl-js#372] [Q&A]
Hi guys, I'm using maplibre-gl inside my react app. after compiling with "react-scripts build" then "serve -s build", then go to the browser it throws "Uncaught ReferenceError: y is not defined". got any idea, thanks

**Comment by sheub** [comment]
This issue is similar to the one described in this mapbox threads: https://github.com/mapbox/mapbox-gl-js/issues/10173

One solution for create-react-app users given in the thread is to update browserlists in package.json to exclude older browsers

browserslist: [
      ">0.2%",
      "not ie 11",
      "not dead",
      "not chrome 49"
    ]

**Reply by tengl** [reply]
Thanks @sheub, you list didn't work, but it pointed me to the right direction, this comment https://github.com/mapbox/mapbox-gl-js/issues/10173#issuecomment-1361872443

Here's what works now.

```json
"browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not ie 11",
      "not chrome < 51",
      "not safari < 10",
      "not android < 51"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
```
